src/Appwrite/Utopia/Response/Model/EdgeSync.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class EdgeSync extends Model
+{
+    public function __construct()
+    {
+
+        $this
+            ->addRule('keys', [
+                'type' => self::TYPE_STRING,
```

Comment:
As mentioned before, we don't need to return the full payload. It is important to mention this is also not the way to declare an array of strings. You can check one of the other examples in the source when it does needed.